### ISSUE TITLE
Handle change of wwpn of a host in ibm_svc_host

### ISSUE THREAD

**Issue by freedge** [ansible-collections/ibm.spectrum_virtualize#9]
##### SUMMARY
Right now most changes in the inventory will be ignored by the module, for example in ibm_svc_host only a change of type is detected and will trigger an actual change command.

I'm opening this ticket in the hope that the module could detect a change of host wwpn and either apply the needed change command, either exit with an error, in either case a run in check mode should report that a change should occur.

Do you plan to add a more generic way to probe for changes?

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
ibm_svc_host and others

##### ADDITIONAL INFORMATION

```yaml

```

**Comment by freedge** [comment]
This is working better with collection ibm.spectrum_virtualize  v1.2.0

to be picky, this module is not idempotent if the wwpn provided is lower case. In fact, if the wwpn is in lowercase, the module will systematically issue an rmhostport command followed by an addhostpost (maybe causing an outage for a fraction of second?)

I modified by playbook so that it now reads:
```
        fcwwpn: "{{ wwpnb | replace(':','' ) | upper }}:{{ wwpna | replace(':','' ) | upper }}"
```

which does the trick, but if the module itself could perform a case insensitive comparison of the wwpn that would be safer.

**Comment by rohitk-github** [comment]
Ok, will analyse this. Have you checked the behavior of SVC commandline in this respect ? Is the wwpn case insensitive in svc ?

**Comment by Shilpi-J** [comment]
Fixed in v1.3.0
